rpistart.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash 
```

Comment:
This needs some explanation of how to invoke it and what it's going to do.  I think you need to pass the name of the `-qemu.img` file?

Reply:
What other parts of it really should be configured externally?

Reply:
Generally a good practice to put the code that reads & validates the input arguments near the top for clarity.